docs/ref/templates/builtins.txt

```diff
@@ -2596,6 +2596,50 @@ slightly different call::
     See :class:`~django.template.backends.jinja2.Jinja2` for information on
     using the ``static`` tag with Jinja2.
 
+.. templatetag:: site_assets
+
+``site_assets``
+~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.2
+
+The ``site_assets`` template tag outputs all site assets defined in installed
+apps :attr:`django.apps.AppConfig.assets`.
+
+Considering this AppConfig definition::
+
+    from django.apps.assets import CSS, JS
+
+    class MyAppConfig(AppConfig):
+        assets = [
+            CSS('css/style.css'),
```

Comment:
Appreciate this pull request is closed, but adding these notes here whilst I think about it. It may help someone in the future. 

I think it would be good to be able to add a folder of assets at a time, in addition to a file at a time. 

In addition to being able to identify within an app I wonder if it's worth while being able to split down even further. For example may have a two JS files, one of which is for a calendar within a form. May wish to only load that asset on a certain page within an app.